runtime/near-evm-runner/Cargo.toml

```diff
@@ -68,3 +68,6 @@ required-features = ["protocol_feature_evm"]
 name = "bench"
 required-features = ["protocol_feature_evm"]
 harness = false
+
+[package.metadata.workspaces]
+independent = true
```

Comment:
I recall there was some mention that we plan to use `cargo-workspaces`. It would be great to have a short document/section in CONRIBUTION.md walking through the flow and common features (like this `package.metadata.workspaces` special Cargo.toml section) :pray: